pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -217,6 +243,10 @@ private static void start() {
 
         // GAE specifies port to use via environment variable
         String appEnginePort = System.getenv(PORT);
+
+        EmbeddedServers.add(
+                EmbeddedServers.Identifiers.JETTY,
+                new AppEngineServerFactory(new JettyServer(), cfg.getString(ConfigFile.PREFERRED_SOURCE_IP_HEADER)));
```

Comment:
Will this fall-back to the default behavior, or log a warning if this configuration parameter is not present or blank?

Reply:
Do we get the `cfg.getString()` thing? That should fail with an exception if it's missing.

Reply:
I assume we should allow this to tolerate no header, and just fall back on the "normal" implementation, right?

Reply:
Yes, that's my intent here- Instead of breaking things if it's missing, fallback to the normal behavior instead.